redshift/config.go

```diff
@@ -34,16 +33,17 @@ func (c *Config) Client() *Client {
 		c.database)
 
 	db, err := sql.Open("postgres", conninfo)
-
 	if err != nil {
-		log.Fatal(err)
-		panic(err)
+		db.Close()
+		return nil, err
```

Comment:
Could wrap err here too with some message, Cannot connect to redshift

Reply:
It doesn't attempt to connect until the first query, so I'm not sure what sort of errors could occur here.

Reply:
Good point, no need so